### PR TITLE
Feature/427 update to angular 16

### PR DESCRIPTION
**Pull request by ds-mmaul**
resolves eclipse-tractusx/traceability-foss#427

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -36,27 +36,27 @@
...
+    "@angular/forms": "16.2.12",
+    "@angular/material": "16.2.13",
+    "@angular/platform-browser": "16.2.12",
+    "@angular/platform-browser-dynamic": "16.2.12",
+    "@angular/router": "16.2.12",
     "@types/lodash-es": "4.17.7",
-    "angular-i18next": "15.0.5",
-    "angular-split": "15.0.0",
+    "angular-i18next": "16.0.0",
+    "angular-split": "^16.2.1",
```
Comment (ds-mwesener) [inline frontend/package.json:52]: Please remove ^

File: frontend/src/app/mocks/services/admin-mock/admin.model.ts [hunk 2]
```diff
@@ -108,3 +108,103 @@ export const getBpnConfig = (): BpnConfigResponse[] => [
     url: 'test.de/BPNL000000TESTKC',
   },
 ];
+
+export const getImportReport = (): any => { return {
```
Comment (ds-mwesener) [inline frontend/src/app/mocks/services/admin-mock/admin.model.ts:112]: What is this here?

**Review by ds-mwesener — CHANGES_REQUESTED** [review]
Please fix

File: frontend/src/app/mocks/services/admin-mock/admin.model.ts (hunk 2 above)
Comment (ds-mmaul) [reply frontend/src/app/mocks/services/admin-mock/admin.model.ts:112]: mock data if u run it locally without BE instance

**Review by ds-mwesener — APPROVED** [review]

**Pull request merged**